addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4195,6 +4195,40 @@ export class OdooEditor extends EventTarget {
             return;
         }
         const anchorNode = selection.anchorNode;
+        const isSelectionInEditable = this.isSelectionInEditable(selection);
+        if (this._isMouseStateActive && selection.isCollapsed && isSelectionInEditable) {
+            const range = getDeepRange(this.editable);
+            const { startContainer, startOffset, endContainer, endOffset } = range;
+            const linkElement = closestElement(startContainer, "a");
+            if (linkElement) {
```

Comment:
You should add a check here if linkElement.textContent startWith and endsWith '\uFEFF' just to be extra careful.